dropwizard-hibernate/pom.xml

```diff
@@ -67,6 +67,14 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
+            <artifactId>caffeine</artifactId>
```

Comment:
Hi @zUniQueX, if you don't want to add an additional dependency, there is a [cache](https://javadoc.io/static/net.bytebuddy/byte-buddy/1.10.2/net/bytebuddy/TypeCache.html) provided by ByteBuddy.

Reply:
I've been thinking about this too but the classes are always loaded by the classloader of the class being proxied, so we can use a normal cache. Caffeine is already pulled in by `dropwizard-jackson`, so this dependency is already present.